pkg/gatewayserver/io/io.go

```diff
@@ -372,8 +372,9 @@ func (c *Connection) ScheduleDown(path *ttnpb.DownlinkPath, msg *ttnpb.DownlinkM
 			rxErrs = append(rxErrs, errRxWindowSchedule.WithCause(err).WithAttributes("window", i+1))
 			continue
 		}
-		settings.Time = nil
-		settings.Timestamp = uint32(time.Duration(em.Starts()) / time.Microsecond)
+		if settings.Time == nil {
```

Comment:
This only works for GPS time synced gateways, hence the original code, which always used fallback to concentrator timestamp.

It's good to use GPS time when we know it's available, but still we must otherwise rely on timestamps, i.e. for absolute time scheduling using RTTs.

So add a check here to see if there's a gateway absolute time sync available